mtp_api/apps/core/management/commands/testserver.py

```diff
@@ -78,22 +77,8 @@ def load_test_data(self, verbosity=1, **kwargs):
         )
 
     @synchronised
-    def create_super_admin(self):
-        try:
-            admin_user = User.objects.get(username='admin')
-        except User.DoesNotExist:
-            admin_user = User.objects.create_superuser(
-                username='admin',
-                email='admin@mtp.local',
-                password='adminadmin',
-                first_name='Admin',
-                last_name='User',
-            )
-        for group in Group.objects.all():
-            admin_user.groups.add(group)
-        give_superusers_full_access()
-
-        self.stdout.write(self.style.SUCCESS('Model creation finished'))
+    def _create_super_admin(self):
+        return create_super_admin(self.stdout, self.style.SUCCESS)
```

Comment:
you still need the `give_superusers_full_access` to be called after this to make sure the new super-admin has access to all client applications. otherwise the standalone test server won't have a usable admin user until something triggers the `load_test_data` management command.

Reply:
the `create_super_admin` function itself totally does the correct thing tho